src/test/java/io/odpf/depot/redis/parsers/RedisHashSetEntryParserTest.java

```diff
@@ -59,9 +60,38 @@ private void redisSinkSetup(String field) throws IOException {
         schemaKey = odpfMessageParser.getSchema(schemaKeyClass, descriptorsMap);
     }
 
```

Comment:
can we add more tests? timestamp? repeated? nonrepeated? 

what about error scenarios?

Reply:
added